<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WannaBecomeARunner</class>
 <widget class="QWidget" name="WannaBecomeARunner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marathon Skills 2017</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(79, 79, 79);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: white;</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white
</string>
        </property>
        <property name="text">
         <string>      MARATHON SKILLS 2017</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>204</width>
          <height>54</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
min-width: 200;
max-width: 200;
min-height: 50;
max-height: 50;</string>
        </property>
        <property name="text">
         <string>Я участвовал ранее</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>54</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>204</width>
          <height>54</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
min-width: 200;
max-width: 200;
min-height: 50;
max-height: 50;</string>
        </property>
        <property name="text">
         <string>Я новый участник</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="lower_pamel" native="true">
     <property name="minimumSize">
      <size>
       <width>960</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(79, 79, 79)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
